genieutils.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b4b3abb..760a912 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -12,11 +12,11 @@ project(genieutils)
+ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake/)
+ #set(CMAKE_CXX_FLAGS "-std=gnu++0x")
+ 
+-#set(Boost_USE_STATIC_LIBS        ON)
++set(Boost_USE_STATIC_LIBS        ON)
+ #set(Boost_USE_STATIC_RUNTIME     ON)
+ set(Boost_USE_MULTITHREADED      ON)
+ 
+-find_library(iconv REQUIRED)
++find_library(iconv iconv REQUIRED)
```

Comment:
Sorry, I am a cmake noob, what does this change do?

Reply:
https://cmake.org/cmake/help/latest/command/find_library.html

I think it just defines it shorthand as it should be so it states the <VAR> and a name.

Reply:
Related https://github.com/SiegeEngineers/auto-mods/issues/1#issuecomment-767336753

Reply:
Actually also essential to find `libiconv` on MSYS/MingW64

Reply:
If we stick to the `format-patch`/`apply-patch` approach and not the script, this should be:

```suggestion
-find_library(iconv REQUIRED)
+find_package(Iconv REQUIRED)
```

Reply:
if that works as well / better, fine by me :shrug:

Reply:
`find_package` should be the preferred method as it's more high-level. As long as that package is deployed with the CMake version. We have to update `cmake_minimum_required(VERSION 3.10)` to `cmake_minimum_required(VERSION 3.11)` though for this change. But it will close #1 as well.

Reply:
Opened Upstream PR https://github.com/Tapsa/genieutils/pull/3 as that's not an issue with us but in general.

Reply:
https://github.com/Tapsa/genieutils/blob/master/CMakeLists.txt#L19 and https://github.com/Tapsa/genieutils/blob/master/CMakeLists.txt#L35 

Which is seems strange to me. Could be that we don't even need to patch this anymore if `Iconv` is required as an upstream package.

Reply:
Probably superseded by https://github.com/Tapsa/genieutils/pull/5

Reply:
Tapsa merged @FranciscoDA @HSZemi